### ISSUE TITLE
Images aren't centered in responsive items in IE11

### ISSUE THREAD

**Issue by bekl** [Semantic-Org/Semantic-UI#4969]
Images aren't centered in responsive items in IE11. 

IE11:

![responsive item ie11](https://cloud.githubusercontent.com/assets/20027950/22517319/04bda622-e8a9-11e6-895c-ec8229dfc382.png)

FF51 (as a comparison how it should be):

![responsive item ff](https://cloud.githubusercontent.com/assets/20027950/22517308/003a5e06-e8a9-11e6-8c0f-ec1101a5e546.PNG)

**Comment by philrykoff** [comment]
According to caniuse, IE11 has a lot of issues (aka bugs) with flexbox.

If you depend on IE11, I would recommend alternative / additional centering methods.

**Comment by bekl** [comment]
Is there a fix/workaround?

**Comment by Banandrew** [comment]
@bekl 

There’s a [community-curated list of flexbox issues and cross-browser workarounds for them](https://github.com/philipwalton/flexbugs) that might help you out.

**Comment by Banandrew** [comment]
Duplicate of #2994.
